.github/dependabot.yml

```
@@ -0,0 +1,33 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/web"
+    schedule:
+      interval: "weekly"
+    groups:
+      web:
+        update-types:
+          - "minor"
+          - "patch"
+
+  - package-ecosystem: "pip"
+    directory: "/api"
+    schedule:
+      interval: "weekly"
+    groups:
+      api:
+        update-types:
+          - "minor"
+          - "patch"
+
+  - package-ecosystem: "docker"
+    directories:
+      - "/web"
+      - "/api"
```

Is this correct? Wouldn't it just be this?
```suggestion
    directories:
      - "/"
```

---

Need to point to where the Dockerfiles are to be able to update the image tags. We are doing this already in another project (void) that uses the template 😊